packages/blitz-rpc/src/index-server.ts

```diff
@@ -37,16 +37,31 @@ const g = getGlobalObject<{blitzRpcResolverFilesLoaded: ResolverFiles | null}>(
   },
 )
 
+const gConfig = getGlobalObject<{
+  blitzRpcResolverConfig: Record<string, ResolverConfig> | null
+}>("__internal_blitzRpcResolverConfig", {
+  blitzRpcResolverConfig: null,
+})
+
 export function loadBlitzRpcResolverFilesWithInternalMechanism() {
   return g.blitzRpcResolverFilesLoaded
 }
 
+async function getResolverConfig(): Promise<Record<string, ResolverConfig> | null | undefined> {
+  if (gConfig.blitzRpcResolverConfig) {
+    return gConfig.blitzRpcResolverConfig
+  }
+}
+
 export function __internal_addBlitzRpcResolver(
   routePath: string,
+  resolverConfig: ResolverConfig,
   resolver: () => Promise<{default?: Resolver}>,
 ) {
   g.blitzRpcResolverFilesLoaded = g.blitzRpcResolverFilesLoaded || {}
   g.blitzRpcResolverFilesLoaded[routePath] = resolver
+  gConfig.blitzRpcResolverConfig = gConfig.blitzRpcResolverConfig || {}
+  gConfig.blitzRpcResolverConfig[routePath] = resolverConfig
   return resolver
```

Comment:
You don't need to add a separate resolverConfig object & loading here, because you can get the config directly from `resolver.config` 

```suggestion
export function __internal_addBlitzRpcResolver(
  routePath: string,
  resolver: () => Promise<{default?: Resolver, config: ResolverConfig}>,
) {
  g.blitzRpcResolverFilesLoaded = g.blitzRpcResolverFilesLoaded || {}
  g.blitzRpcResolverFilesLoaded[routePath] = resolver
  return resolver
```

Reply:
thanks, @flybayer. I updated with this logic. I was pretty confused about what the resolver function was returning, which is why I used the same logic I had used for the client for the server also...